cmd/cstor-pool-mgmt/controller/pool-controller/new_pool_controller.go

```diff
@@ -73,7 +73,8 @@ func NewCStorPoolController(
 	// obtain references to shared index informers for the cStorPool resources
 	cStorPoolInformer := cStorInformerFactory.Openebs().V1alpha1().CStorPools()
 
-	openebsScheme.AddToScheme(scheme.Scheme)
+	err := openebsScheme.AddToScheme(scheme.Scheme)
+	glog.Errorf("failed to add to scheme: error {%v}", err)
```

Comment:
Should we return error here? cc @sonasingh46 
@shubham14bajpai lets talk to @sonasingh46 for error handling in this case

Reply:
It should check if `err != nil`

Reply:
Done